### DISCUSSION TITLE
Copilot Chat enable issue (with access) + install issue in WSL

### DISCUSSION THREAD

**Discussion by eek** [community/community#55019] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi there, even though I have access to the Technical Preview (confirmed by email & website) I have 2 issues with VSCode:

1. Even though the extension is installed, I can't seem to be able to bring the chat window up, there's no icon in the extension bar, and CTRL+I doesn't work. What am I doing wrong?

2. I also tried installing and running it in WSL. But there, I can't seem to install it. After installing, it says that "This extension is disabled in this workspace because it is defined to run in the Remote Extension Host. Please install the extension in 'WSL: Ubuntu' to enable.". 
The WSL Extension is installed, and the Github Copilot Nightly says `Extension is enabled on 'WSL: Ubuntu'`
<img width="558" alt="image" src="https://github.com/community/community/assets/2953026/0413e6b1-0ae3-4403-89fc-aabacba454cd">

And if I hit "Install in WSL: Ubuntu" it errors with "Can't install release version of 'github.copilot-chat' extension because it has no release version." even though I have the Pre-Release already installed and can't seem to find a way to make it install the pre-release not the release.

<img width="891" alt="image" src="https://github.com/community/community/assets/2953026/e388fbff-b34f-4720-ae6b-555fd4883d0b">

--------
Versions:
```
Version: 1.78.1 (user setup)
Commit: 6a995c4f4cc2ced6e3237749973982e751cb0bf9
Date: 2023-05-04T09:48:08.683Z
Electron: 22.5.1
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22624
Sandboxed: Yes
```

**Comment by davecheney** [comment, marked as answer]
@eek the chat extension is currently only compatible with vscode-insiders. Please consult the installation instructions in your welcome email.

**Comment by patricktree** [comment]
The answer provided by @davecheney did not help me to get the extension into the WSL environment, but I could solve it by running this command in a WSL terminal:

```
code-insiders --install-extension github.copilot-chat@prerelease
```

**Reply by bHimes** [reply]
YES! this also worked inside a Dev Container in vscode on linux

**Reply by jeffscottward** [reply]
not working on mac. getting command not found: code-insiders

**Reply by iamSymbiote** [reply]
@jeffscottward you need to use the "insider" version of the VSCode mate. I tried but can't see the copilot chat tab activity monitor **on normal vs code**, but tried on **vscode insider** app with a copilot nightly, worked like a charm.

**Reply by jeffscottward** [reply]
I actually just got it working. There was something weird going on with the insiders download. Even though it was called insider the icon was the same blue. I was clearly the stable build. I went to re-download it and saw the icon change color and its now a fresh build without any extensions or anything so I redid the logins and extensions and it's good now.


> @jeffscottward you need to use the "insider" version of the VSCode mate. I tried but can't see the copilot chat tab activity monitor **on normal vs code**, but tried on **vscode insider** app with a copilot nightly, worked like a charm.

**Reply by iamSymbiote** [reply]
oh good to hear that, I suppose the insider logo will be green tho.
Also it's looking and working sharp, can't wait to dig in to the chat :)

**Reply by Sirfrummel** [reply]
I installed vscode insiders, but I'm not getting code-insiders command available in WSL.  I was able to manually get a WSL window connected in VSCode Insiders but my copilot chat isn't showing there (where it was previously in insiders).  any ideas?

**Reply by Sirfrummel** [reply]
ahh, got it!  was able to run that command in VS Code's WSL built-in terminal. code-insiders command worked there.

**Reply by oldbuilding** [reply]
If you don't have insiders build, this works too.

`code --install-extension github.copilot-chat@prerelease`

Edit: the installation appears to have worked with non-insiders vs code, but the extension doesn't.  I'll just switch to vs code insiders.

**Comment by jeffscottward** [comment]
@davecheney will we get back the days we lose out of the 60 days we are allotted?

Im on Mac M1 Max. 

Have insiders
Have nighly
Have chat pre-release
get - nothing.

*VS Code Insiders*
```
Version: 1.78.2 (Universal)
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:44:45.204Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: Yes
```

*Co-pilot Nightly*
```
v1.86.104
```

*Co-pilot Chat*
```
v0.1.2023051803 - Pre-Release
```
